packages/react-components/react-menu/src/selectable/types.ts

```diff
@@ -28,16 +28,6 @@ export type MenuItemSelectableProps = {
  * State for selectable menu items
  */
 export type MenuItemSelectableState = MenuItemSelectableProps & {
-  /**
-   * Checked items for a value with `name`
-   */
-  checkedItems: string[];
-
-  /**
-   * Callback when checked items changes for a given value with `name`
-   */
-  onCheckedValueChange: (e: React.MouseEvent | React.KeyboardEvent, name: string, checkedItems: string[]) => void;
-
```

Comment:
Can we safely remove these from the types without causing a breaking change?

Reply:
@ling1726